tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/debugger/breakpoint/BallerinaBreakPointType.java

```diff
@@ -63,6 +62,13 @@ private static boolean isLineBreakpointAvailable(@NotNull VirtualFile file, int
         }
         Checker canPutAtChecker = new Checker();
         XDebuggerUtil.getInstance().iterateLine(project, document, line, canPutAtChecker);
+
+        // Used to identify line comments, which cannot be correctly processed by the checker.
+        if (document.getText(new TextRange(document.getLineStartOffset(line), document.getLineEndOffset(line)))
```

Comment:
I think this will not match if the comment has leading spaces.

Eg -

```ballerina
    // some comment
```

Reply:
Fixed it by trimming whitespaces before the check.